presto-main/src/main/java/io/prestosql/server/PluginClassLoader.java

```diff
@@ -16,8 +16,6 @@
 import com.google.common.collect.ImmutableList;
```

Comment:
Are there any changes between java 9 and java 11 in regards to platform class loader? Was `getPlatformClassLoader()` method removed? Is there different model of classloaders?